File: beginners/2-problemset-condicionais/C++/2956-derivada-13-variaveis/derivada_13_variaveis.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
    long double P, T, A;

    cin >> P >> T;

    A = (P * T)/2;

    cout << "Concluimos que, dado o limite da entrada, a resposta seria:  y = f(x) = " << fixed << setprecision(5) << A << "." << endl;

    return 0;
}

File: beginners/2-problemset-condicionais/Python/2956-derivada-13-variaveis/derivada_13_variaveis.py
base, altura = input().split()
base = float(base)
altura = float(altura)

area_triangulo = (base * altura) / 2

print(
    f'Concluimos que, dado o limite da entrada, a resposta seria:  y = f(x) = {area_triangulo:.5f}.')
